public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Mel's hack - the missing bits"
```

Comment:
- Should be: "Mel's Hack - The Missing Bits"
- Change filename so URL will reflect this title

Reply:
Should I do it twice?